components/movementsensor/imuvectornav/imu.go

```diff
@@ -30,32 +30,33 @@ type AttrConfig struct {
 	SPI   string `json:"spi"`
 	Speed *int   `json:"spi_baud_rate"`
 	Pfreq *int   `json:"polling_freq_hz"`
-	CSPin string `json:"cs_pin"`
+	CSPin string `json:"chip_select_pin"`
```

Comment:
this is a breaking change, let's do this separately and bundle with other breaking changes

Reply:
I don't think anyone is using a vectornav except Nicolas. Might be okay?

Reply:
Yes i am the only one so i vote we should do it :)